npm/design-system/src/components/searchInput/SearchInput.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react'
+import { mount } from '@cypress/react'
+
+import { SearchInput } from './SearchInput'
+import { useCallback, useState } from 'react'
+
+describe('SearchInput', () => {
+  const StatefulWrapper: React.FC<{onInput?: (input: string) => void}> = ({ onInput }) => {
+    const [value, setValue] = useState('')
+
+    const memoedOnInput = useCallback((input: string) => {
+      setValue(input)
+      onInput?.(input)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
How come you don't pass the deps to this hook?

Reply:
This is a test and `onInput` should not be changing